torch_geometric/nn/models/huggingface_hub_wrapper.py

```diff
@@ -0,0 +1,124 @@
+import json
+import logging
+import os
+from pathlib import Path
+from typing import Dict, Optional
+
+import requests
+from huggingface_hub import (
```

Comment:
I guess you would need to wrap this around `try` and `except ImportError` to treat `huggingface_hub` correctly as an optional dependency.